food/examples/4.json

```diff
@@ -0,0 +1,21 @@
+  "source": "foodista",
+  "added": "2024-05-07T18:51:12.319767",
+  "created": "2007-12-11T00:00:00",
+  "metadata": {
+    "license": "Creative Commons - Attribution - https://creativecommons.org/licenses/by/3.0/",
+    "url": "https://foodista.com/blog/2007/12/11/theres-salt-and-theres-salt",
+    "authors": [
+      [
+        "Gâteau au Foie ...",
```

Comment:
Some of these comments seem to be some kind of truncated version of articles posted elsewhere (probably for backlinks?) Should we try to omit those? The legit comments may be worth keeping.